addons/orchestrator/templates/cmpd-raft.yaml

```diff
@@ -18,19 +18,21 @@ spec:
 
   lifecycleActions:
     roleProbe:
-      exec:
-        command:
-          - /bin/bash
-          - -c
-          - |
-            role=$(curl -s http://127.0.0.1:3000/api/leader-check)
-            if [[ $role == "\"OK\"" ]]; then
-              echo -n "primary"
-            elif [[ $role == "\"Not leader\"" ]]; then
-              echo -n "secondary"
-            else
-              echo -n ""
-            fi
+      builtinHandler: custom
+      customHandler:
```

Comment:
Add time settings for this action, like:
      periodSeconds: 1
      initialDelaySeconds: 60
      timeoutSeconds: 10

Reply:
it is good to set them as they are best recommendations for orchestrator.

Reply:
@nayutah talk is cheap, show me the data